suites/quincy/baremetal_pipeline/tier-1_rgw.yaml

```diff
@@ -73,7 +73,41 @@ tests:
       module: test_client.py
       name: configure client
 
-  # Testing stage
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            cephadm: true
+            commands:
+              - "radosgw-admin realm create --rgw-realm india --default"
+              - "radosgw-admin zonegroup create --rgw-realm india --rgw-zonegroup shared --endpoints http://{node_ip:folio15}:8080,http://{node_ip:folio16}:8080,http://{node_ip:folio17}:8080,http://{node_ip:folio18}:8080,http://{node_ip:folio19}:8080 --master --default"
+              - "radosgw-admin zone create --rgw-realm india --rgw-zonegroup shared --rgw-zone primary --endpoints http://{node_ip:folio15}:8080,http://{node_ip:folio16}:8080,http://{node_ip:folio17}:8080,http://{node_ip:folio18}:8080,http://{node_ip:folio19}:8080 --master --default"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "radosgw-admin user create --uid=repuser --display_name='Replication user' --access-key test123 --secret test123 --rgw-realm india --system"
+              - "radosgw-admin zone modify --rgw-realm india --rgw-zonegroup shared --rgw-zone primary --access-key test123 --secret test123"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "ceph config set client.rgw.* rgw_realm india"
+              - "ceph config set client.rgw.* rgw_zonegroup shared"
+              - "ceph config set client.rgw.* rgw_zone primary"
+              - "ceph orch restart rgw.shared.pri.8080"
+        ceph-sec:
+          config:
+            cephadm: true
+            commands:
+              - "sleep 120"
+              - "radosgw-admin realm pull --rgw-realm india --url http://folio15:8080 --access-key test123 --secret test123 --default"
+              - "radosgw-admin period pull --url http://folio15:8080 --access-key test123 --secret test123"
+              - "radosgw-admin zone create --rgw-realm india --rgw-zonegroup shared --rgw-zone secondary --endpoints http://{node_ip:folio20}:8080,http://{node_ip:folio21}:8080,http://{node_ip:folio22}:8080,http://{node_ip:folio23}:8080,http://{node_ip:folio24}:8080 --access-key test123 --secret test123"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "ceph config set client.rgw.* rgw_realm india"
+              - "ceph config set client.rgw.* rgw_zonegroup shared"
+              - "ceph config set client.rgw.* rgw_zone secondary"
+              - "ceph orch restart rgw.shared.sec.8080"
+      desc: Setting up RGW multisite replication environment
+      module: exec.py
+      name: setup multisite
+      polarion-id: CEPH-10362
 
```

Comment:
@udaysk23 I think these changes have to be done in https://github.com/red-hat-storage/cephci/blob/master/suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml
as this suite is renamed as tier-1_rgw_ms.yaml in this PR https://github.com/red-hat-storage/cephci/pull/2777/files#diff-8f1432e03f47da038b0d33e03ae86ebcd456161e0be1aaf81d69e77d8cb20411